.github/workflows/deploy-docs.yml

```diff
@@ -63,7 +63,6 @@ jobs:
 
   deploy-preview:
     runs-on: ['ubuntu-latest']
-    needs: [garnix]
```

Comment:
This will result in the package being built on the GH runner

Reply:
Also, if we're committing to that, we should do it for all deployments

Reply:
See the chat. We want deployment URLs asap, even if other checks still fail.